src/components/Layout/Layout.tsx

```diff
@@ -12,7 +17,14 @@ const Layout = () => {
                     flex: 1,
                     background: (theme) => theme.palette.background.secondary,
                 }}
-            />
+            >
+                <Routes>
+                    <Route path="/" element={<Home />} />
+                    <Route path="devices" element={<Devices />} />
+                    <Route path="community" element={<Community />} />
+                    <Route path="*" element={<NotFound />} />
+                </Routes>
```

Comment:
I think that those Routes should be placed somewhere in App.tsx (or some Routes.tsx file) and the Layout should be applied like in the following code:
```
<Route element={<Layout />}>
    <Route path="/" element={<Home />} />
    <Route path="devices" element={<Devices />} />
    ...
 </Route>
```
It would be probably easier to extend, if we had more Layouts (e.g. Login/Register page).
See https://reactrouter.com/docs/en/v6/getting-started/concepts#layout-routes

Reply:
Changed